modules/tinymce/src/themes/silver/main/ts/ui/general/Notification.ts

```diff
@@ -14,7 +14,7 @@ import { Arr, Optional } from '@ephox/katamari';
 
 import { TranslatedString, Untranslated } from 'tinymce/core/api/util/I18n';
 
-import { get as getIcon, getFirst, IconProvider } from '../icons/Icons';
+import { addFocusableBehaviour, get as getIcon, getFirst, IconProvider } from '../icons/Icons';
```

Comment:
Can this please be changed to use `* as Icons` and update the references below? It's more consistent and helps keep the context for where it's used below.

Example:
```
import * as Icons from '../icons/Icons';
```